File: example-databinding/src/main/java/com/xwray/groupie/example/databinding/item/SmallCardItem.java
package com.xwray.groupie.example.databinding.item;

public class SmallCardItem extends CardItem {

    public SmallCardItem() {
        super();
    }

    public SmallCardItem(CharSequence text) {
        super(text);
    }

    @Override public int getSpanSize(int spanCount, int position) {
        return spanCount / 3;
    }
}

File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/item/SmallCardItem.kt
package com.xwray.groupie.example.viewbinding.item

import androidx.annotation.ColorInt

open class SmallCardItem : CardItem {
    constructor(@ColorInt colorInt: Int) : super(colorInt)
    constructor(@ColorInt colorInt: Int, text: CharSequence) : super(colorInt, text)

    override fun getSpanSize(spanCount: Int, position: Int): Int = spanCount / 3
}
